README.md

```diff
@@ -37,13 +37,14 @@ install Knative with its prerequisites.
 
 ## Documentation
 
-Start with the [user's guide](docs/README.md) to learn more. You can read about
+Refer [user's guide](docs) to learn more. You can read about
```

Comment:
```suggestion
Refer to the [user's guide](docs) to learn more. You can read about
```

Reply:
And let's keep the `docs/README.md` as a landing page and table of content maybe. 

Let's have a pointer back to knative.dev site, cmd references, and the following:

```markdown
## More information on `kn`:

- [Workflows](workflows/README.md)
- [Operations](operations/README.md)
- [Traffic Splitting](traffic/README.md)
```

Reply:
Brought back the relevant content from existing `docs/README.md` and added a link to knative.dev client docs.